custodia.spec

```diff
@@ -169,6 +169,7 @@ mkdir -p %{buildroot}/%{_unitdir}
 mkdir -p %{buildroot}/%{_tmpfilesdir}
 mkdir -p %{buildroot}/%{_localstatedir}/lib/custodia
 mkdir -p %{buildroot}/%{_localstatedir}/log/custodia
+mkdir -p %{buildroot}/%{_localstatedir}/run/custodia
```

Comment:
We have tmpfiles.d support. The line is required despite tmpfiles.d support, https://fedoraproject.org/wiki/Packaging:Tmpfiles.d#Example_spec_file